SUBSTR and LEFT tests failing with Go 1.11

Notice, they work correctly with Go 1.12, but with Go 1.11:
```
--- FAIL: TestSUBSTRBadOffset (0.00s)
        Error Trace:    vm_test.go:649
        Error:          Not equal: 
                        expected: true
                        actual  : false
        Test:           TestSUBSTRBadOffset
--- FAIL: TestSUBSTRBadLen (0.00s)
        Error Trace:    vm_test.go:659
        Error:          Not equal: 
                        expected: true
                        actual  : false
        Test:           TestSUBSTRBadLen
--- FAIL: TestLEFTBadLen (0.00s)
        Error Trace:    vm_test.go:694
        Error:          Not equal: 
                        expected: true
                        actual  : false
        Test:           TestLEFTBadLen
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -3,11 +3,6 @@ orbs:
   codecov: codecov/codecov@1.0.5
 
 executors:
-  go1_11:
-    docker:
-      - image: circleci/golang:1.11
-        environment:
-          GO111MODULE: "on"
   go1_12:
     docker:
       - image: circleci/golang:1.12
@@ -53,14 +48,6 @@ jobs:
       - run:
           name: go-vet
           command: go vet ./...
-
-  test_1_11:
-    working_directory: /go/src/github.com/CityOfZion/neo-go
-    executor: go1_11
-    steps:
-      - checkout
-      - gomod
-      - run: go test -v -race ./...
 
   test_1_12:
     working_directory: /go/src/github.com/CityOfZion/neo-go
@@ -120,10 +107,6 @@ workflows:
             tags:
               only: v/[0-9]+\.[0-9]+\.[0-9]+/
       - lint:
-          filters:
-            tags:
-              only: v/[0-9]+\.[0-9]+\.[0-9]+/
-      - test_1_11:
           filters:
             tags:
               only: v/[0-9]+\.[0-9]+\.[0-9]+/

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,5 @@
 language: go
 go:
-  - 1.11.x
   - 1.12.x
 env:
   - GO111MODULE=on

--- a/README.md
+++ b/README.md
@@ -31,7 +31,7 @@ A complete toolkit for the NEO blockchain, including:
 
 ## Installation
 
-Go: 1.11+
+Go: 1.12+
 
 Install dependencies.
 

